src/main/kotlin/org/usfirst/frc/team4099/robot/Robot.kt

```diff
@@ -88,7 +105,21 @@ class Robot : IterativeRobot() {
 
     override fun teleopPeriodic() {
         try {
-
+            if (intake.up && controlboard.lowerIntake) {
+                intake.up = false
+                println("Lowering intake")
+            } else if (!intake.up && controlboard.lowerIntake) {
+                intake.up = true
+                println("Raising intake")
+            }
+
+            intake.intakeState = when {
```

Comment:
What does this when block do? Setting intake speed should be the same as changing the intake position (like above) and be handled using if blocks. "when" only takes one argument and therefore can't work for multiple controls